### PR TITLE
Fix `ArgumentOutOfRange: StartIndex cannot be less than zero`

### PR DESCRIPTION
**Pull request by duncansmart**
... for files without extensions such as `LICENSE`

### CODE REVIEW

File: T4MVCHostMvcApp/T4MVC Files/T4MVC.tt [hunk 1]
```diff
@@ -1298,7 +1298,7 @@ PopIndent();
 
 if (!settings.ExcludedStaticFileExtensions.Any(extension => projectItem.Name.EndsWith(extension, StringComparison.OrdinalIgnoreCase))) {
 
-    var extension = projectItem.Name.Substring(projectItem.Name.LastIndexOf("."));
+    var extension = Path.GetExtension(projectItem.Name.LastIndexOf("."));
```
Comment (davidebbo) [inline T4MVCHostMvcApp/T4MVC Files/T4MVC.tt]: I assume you mean `Path.GetExtension(projectItem.Name)`? :)

Comment (duncansmart) [inline T4MVCHostMvcApp/T4MVC Files/T4MVC.tt]: YES! :-/ No Idea how I managed to commit that - thought I'd copied the whole line over from my working project - will resubmit ...

**Comment by duncansmart** [comment]
OK, redone the patch!

**Comment by davidebbo** [comment]
Thanks, it's in 3.16.5.

**Pull request merged**